include/performance_test_fixture/performance_test_fixture.hpp

```diff
@@ -16,10 +16,11 @@
 #define PERFORMANCE_TEST_FIXTURE__PERFORMANCE_TEST_FIXTURE_HPP_
 
 #include <benchmark/benchmark.h>
-#include <osrf_testing_tools_cpp/memory_tools/memory_tools.hpp>
 
 #include <atomic>
 
+#include <osrf_testing_tools_cpp/memory_tools/memory_tools.hpp>
```

Comment:
Should we change to double-quotes here?

Reply:
I don't know. cpplint is happy either way.

Reply:
Well, maybe if we use double-quotes both old and new versions of cpplint will be happy. I'll give it a try.